src/controllers/beneficial.owner.individual.controller.ts

```diff
@@ -30,6 +41,10 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     data[DateOfBirthKey] = prepareData(req.body, DateOfBirthKeys);
     data[StartDateKey] = prepareData(req.body, StartDateKeys);
 
+    data[BENEFICIAL_OWNER_NOC] = [].concat(req.body[BENEFICIAL_OWNER_NOC]);
```

Comment:
To be in the safe side, because we do not have yet validation in place, I would add a check before the concat. Something like 
```
data[BENEFICIAL_OWNER_NOC] = (data[BENEFICIAL_OWNER_NOC]) ? [].concat(data[BENEFICIAL_OWNER_NOC]) : [];
data[TRUSTEE_NOC] = (data[TRUSTEE_NOC]) ? [].concat(data[TRUSTEE_NOC]) : [];
data[NON_LEGAL_FIRM_NOC] = (data[NON_LEGAL_FIRM_NOC]) ? [].concat(data[NON_LEGAL_FIRM_NOC]) : [];
```